deploy/operator.yaml

```diff
@@ -79,6 +79,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: NODE_NAME
```

Comment:
we need to sync this to the bundle and the manifests folder to have this part of OLM deployment